commands/function.ts

```diff
@@ -1,16 +1,16 @@
-// @ts-nocheck
```

Comment:
We're doing these command buckets differently now. We have some custom types that we're using to get some more value out of TS. Here's an example: https://github.com/HubSpot/hubspot-cli/blob/main/commands/project.ts